Parse-Dashboard/index.js

```diff
@@ -41,26 +35,66 @@ packageJson('parse-dashboard').then(latestPackage => {
   }
 });
 
-app.get('/parse-dashboard-config.json', function(req, res) {
-  jsonFile(configFile)
-  .then(config => {
-    config.data.apps.forEach((app) => {
-      if (!app.appName) {
-        return res.send({ success: false, error: 'An application is misconfigured, appName is required' });
-      }
-    });
-    var response = {
+const port = program.port || process.env.PORT || 4040;
+const allowInsecureHTTP = program.allowInsecureHTTP || process.env.PARSE_DASHBOARD_ALLOW_INSECURE_HTTP;
+
+let configFile = null;
+let configFromCLI = null;
+if (!program.config) {
+  if (!program.appId || !program.masterKey || !program.serverURL) {
+    console.log('You must provide either a config file or an app ID, Master Key, and server URL. See parse-dashboard --help for details.');
+    process.exit(4);
+  }
+  configFromCLI = {
+    data: {
+      apps: [
+        {
+          appId: program.appId,
+          serverURL: program.serverURL,
+          masterKey: program.masterKey,
+          appName: program.appName,
+        }
+      ]
+    }
+  }
+} else {
+  configFile = program.config || path.join(__dirname, 'parse-dashboard-config.json');
```

Comment:
Are we going to deprecate that file? That doesn't really seem necessary. At any rate, we can do that in a future version.